aioriak/client.py

```diff
@@ -338,6 +338,8 @@ def bucket(self, name, bucket_type='default'):
 
     async def get_keys(self, bucket):
         '''
+            coroutine
```

Comment:
This `sphinxcontrib-asyncio` is used, does this note ever need? Also, it seems like incomplete.